docs/reference/field-ref.asciidoc

```diff
@@ -102,22 +104,23 @@ These fields can be mapped to display additional network data in the {es-sec-app
 [discrete]
 ==== DNS query fields
 
-These fields must be mapped to display DNS data in the {es-sec-app}:
+{es-sec} relies on these fields to analyze and display DNS data:
 
 * `dns.question.name`
 * `dns.question.registered_domain`
 
-This field can be mapped to display additional DNS data in the {es-sec-app}:
+{es-sec} may also use this field to display DNS data in the (es-sec-app):
 
 * `dns.question.type`
+
 +
 NOTE: If you want to be able to filter out PTR records, make sure relevant
-events populate the `dns.question.type` field with a value of `PTR`.
+events are populated with `dns.question.type:ptr` field values of `PTR`.
```

Comment:
You might want to check on this, but this might be redundant, because I think `dns.question.type:ptr` is just another way of writing the pairing of the field `dns.question.type` and the value `ptr` (or `PTR`)? So maybe it could just be this, or something similar:
```suggestion
events are populated with `dns.question.type:ptr`.
```

Reply:
Thanks, I'll follow up on this.